libdeno/BUILD.gn

```diff
@@ -16,23 +13,35 @@ config("deno_config") {
   }
 }
 
-v8_static_library("v8") {
-  public_deps = [
-    "//build/win:default_exe_manifest",
+config("win_crt_agnostic") {
+  if (is_win) {
+    cflags = [ "/Zl" ]
+  }
+}
```

Comment:
Add this config to deno_config ?

Reply:
It needs to be applied to libdeno but not to deno.exe.